frontend/src/router/index.js

```diff
@@ -39,14 +39,12 @@ const router = new VueRouter({
   routes
 })
 
-
-
 router.beforeEach((to, from, next) => {
   const currentUser = firebase.auth().currentUser;
   const requiresAuth = to.matched.some(record => record.meta.requiresAuth);
 
   if (requiresAuth && !currentUser) next('login');
-  else if (!requiresAuth && currentUser) next('home');
+  //else if (!requiresAuth && currentUser) next('home');
```

Comment:
As a practice, we never use the commented code.
If possible, please remove the line

Reply:
Ok, will remove that.